---
# pipe frame publisher node

frames:
  start_frame_name: "pipe_mission_start_link"
  bootstrap_frame_name: "pipe_bootstrap_link"
  aligned_start_name: "pipe_start_aligned"

plan:
  default_z_offset_m: -2.0

tf:
  odom: "odom"
  base_link: "taluy/base_link"
  camera: "camera_optical_link"

# pipe follower visual servo node

max_lin: 0.8
max_ang: 1.2

pid_yaw:
  kp: 0.35
  ki: 0.01
  kd: 0.04

lat_to_deg_gain: 0.01

bootstrap_secs: 5.0

roi_top_frac: 0.0
num_segments: 5
